composer/algorithms/ema/ema.py

```diff
@@ -262,7 +262,10 @@ def apply(self, event: Event, state: State, logger: Logger) -> None:
             # Update the ema model
             compute_ema(state.model, self.ema_model, smoothing=self.smoothing)
 
-        if event == Event.EVAL_START and self.ema_weights_active is False:
+        if event == Event.EVAL_START:
+            # Verify that the ema params are on the correct device.
+            # Needed to ensure doing eval before training can resume correctly.
+            self.ema_model.move_params_to_device(destination_model=state.model)
```

Comment:
Can we please call this on PREDICT_START too?

Reply:
Done